module-apps/application-settings/models/bluetooth/BluetoothSettingsModel.hpp

```diff
@@ -40,6 +40,7 @@ class BluetoothSettingsModel
     auto isDeviceConnecting() -> bool;
     auto getStatus() const -> const BluetoothStatus;
     auto isDeviceListEmpty() const -> bool;
+    Devicei pinRequestor;
```

Comment:
why need another one as there is devicei vector below?

Reply:
This are two different elements:
- this is the device which requested for pin
- the vector is list of elements in scan and is a mirror of data in Bluetooth service